addons/website/static/src/xml/website.xml

```diff
@@ -59,27 +59,31 @@
     </t>
 
     <t t-name="website.dependencies">
-        <t t-if="dependencies">
-            <p class="text-warning">
-              <strong>Warning!</strong> Don't forget to update all links referring to this page:
-            </p>
-            <t t-set='tid' t-value='0'/>
-            <div t-foreach="_.keys(dependencies)" t-as="type" class="mb16">
-                <a class="collapsed fa fa-caret-right" data-toggle="collapse" t-att-href="'#collapseDependencies'+tid" aria-expanded="false" t-att-aria-controls="'collapseDependencies'+tid">
-                  <t t-esc="type"/>:&amp;nbsp;
-                  <span class="text-muted"><t t-esc="dependencies[type].length"/> found(s)</span>
-                </a>
-                <div t-att-id="'collapseDependencies'+tid" class="collapse" aria-expanded="false" >
-                  <ul>
-                      <li t-foreach="dependencies[type]" t-as="error">
-                          <a t-if="!_.contains(['', '#', false,], error.link)" t-att-href="error.link">
-                              <t t-raw="error.text"/>
-                          </a>
-                          <t t-if="_.contains(['', '#', false,], error.link)" t-raw="error.text"/>
-                      </li>
-                  </ul>
+        <t t-if="_.keys(dependencies).length > 0">
+            <div class="panel panel-warning">
```

Comment:
If you put the `t-if` on the `div` instead of a new `t`, you gain one indentation.